src/pybind/mgr/cephadm/tests/test_services.py

```diff
@@ -953,14 +943,17 @@ def gen_cert(host, addr):
 
                 scrape_configs:
                   - job_name: 'ceph'
-                    scheme: https
-                    tls_config:
-                      ca_file: root_cert.pem
-                    honor_labels: true
                     relabel_configs:
+                    - source_labels: [__address__]
+                      target_label: cluster
+                      replacement: fsid
                     - source_labels: [instance]
                       target_label: instance
                       replacement: 'ceph_cluster'
+                    scheme: https
+                    tls_config:
+                      ca_file: root_cert.pem
```

Comment:
we have to double check the implications of this root_cert usage .. as we are now using mTLS